packages/lesswrong/themes/siteThemes/alignmentForumTheme.ts

```diff
@@ -98,46 +98,11 @@ export const alignmentForumTheme: SiteThemeSpecification = {
       },
     },
     overrides: {
-      Header: {
-        titleLink: {
-          top: 0
-        }
-      },
       MuiTooltip: {
         tooltip: {
           fontSize: 13
         }
       },
-      PostsVoteDefault: {
-        voteScores: {
-          fontVariantNumeric: "lining-nums",
-        }
-      },
-      SectionTitle: {
```

Comment:
I don't see the `SectionTitle` override styles being put into the corresponding component

Reply:
The classes named in the override style don't exist anymore, so there was nowhere to transfer those styles to (and they had no effect).